rate/static/js/src/rate.js

```diff
@@ -0,0 +1,60 @@
+/* Javascript for RateXBlock. */
+// Work-around so we can log in edx-platform, but not fail in Workbench
+if (typeof Logger === 'undefined') {
+    var Logger = {
+        log: function(a, b) { 
+	    console.log("<<Log>>"); 
+	    console.log(a);
+	    console.log(b);
+	    console.log("<</Log>>"); 
+	}
+    }
+}
+
+
+function RateXBlock(runtime, element) {
+    var feedback_handler = runtime.handlerUrl(element, 'feedback');
+
+    $(".rate_submit_feedback", element).click(function(eventObject) {
+	freeform = $(".rate_freeform_area", element).val();
```

Comment:
Thank you. Woah -- that caught a lot of stuff. I hadn't used jshint before. Very helpful tool and suggestion.